deployments/qat_dpdk_app/crypto-perf/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+nameSuffix: -crypto-perf
+bases:
+- ../dpdk-test/
```

Comment:
I slept overnight and decided to move dpdk-test, crypto-perf and compress-perf to a dir called patches/ to indicate the overlays are not usable as is.